utils.go

```diff
@@ -351,3 +351,21 @@ func InvokeParsableWriter(writer serialization.ParsableWriter, parsable serializ
 	}
 	return nil
 }
+
+// CopyMap returns a copy of map[string]string
+func CopyMap[T comparable, R interface{}](items map[T]R) map[T]R {
+	result := make(map[T]R)
+	for idx, item := range items {
+		result[idx] = item
+	}
+	return result
+}
+
+// CopyStringMap returns a copy of map[string]string
+func CopyStringMap(items map[string]string) map[string]string {
```

Comment:
what's the value of that specialized method? could it use the other implementation?

Reply:
ah the linker taking forever, that makes sense, thanks for calrifying.